packages/web3-eth/test/unit/format_transaction.test.ts

```diff
@@ -0,0 +1,96 @@
+import { ValidTypes } from 'web3-utils';
+import { formatTransaction } from '../../src/eth_tx';
+import {
+	bigIntTransaction,
+	hexStringTransaction,
+	numberStringTransaction,
+	numberTransaction,
+} from '../fixtures/format_transaction';
+
+describe('formatTransaction', () => {
+	it.skip('should call override method', () => {
+		// const overrideFunction = jest.fn();
+		// formatTransaction(hexStringTransaction, ValidTypes.Number, overrideFunction);
+		// expect(overrideFunction).toHaveBeenCalledWith(hexStringTransaction);
+	});
```

Comment:
If this test is not necessary, then we can remove it.

Reply:
The override function is necessary, just not implemented. There are `TODO`s for this already